test/utilities.js

```diff
@@ -147,6 +147,20 @@ const generateMerkleTree = memoize(_generateMerkleTree, {
   strategy: memoize.strategies.variadic
 })
 
+/**
+ * Partitions arrary into chunks of size spacing
+ * @param input: Array
+ * @param spacing: int
+ * @returns {Array}
+ */
+function splitArray(input, spacing) {
```

Comment:
nit: maybe call splitArrayIntoChunks to make it more clear what this function does without reading its docs?

ubernit: for those who love concise recursive (but not so easy to read) code:

```
return [input.slice(0, spacing)].concat(splitArray(input.slice(spacing), spacing));
```

Reply:
Actually, I would prefer to call this `partitionArray`. 

Does your one liner actually do the same thing? I wouldn't mind using that (but only if I can understand it).

Reply:
partitionArray sounds good to me. And screw the one liner, i usually prefer easy to read code as well.